### PR TITLE
Release

### PR DESCRIPTION
**Pull request by 2opremio**
* Update Weave Net to 1.8.1
* Update Weave Scope to the upcoming 1.1.0 release (https://github.com/weaveworks/scope/pull/2033)
* Update base ECS images (including new Ohio region)
* Add documentation for new actions required for Scope ECS views

Depends on  https://github.com/weaveworks/scope/pull/2033 (which is turn depends on https://github.com/weaveworks/scope/pull/2026 )

### CODE REVIEW

File: aws/ecs/cloudformation-larger-app.json [hunk 1]
```diff
@@ -500,6 +503,9 @@
                     "ecs:ListClusters",
                     "ecs:ListContainerInstances",
                     "ecs:DescribeContainerInstances",
+		    "ecs:ListServices",
```
Comment (errordeveloper) [inline aws/ecs/cloudformation-larger-app.json]: indentation...

File: aws/ecs/README.md [hunk 2]
```diff
@@ -18,24 +18,25 @@ not remove it and respect the format! -->
...
 |----------------|--------------|
-| us-east-1      | ami-3fc1b928 |
-| us-west-1      | ami-4c2a642c |
-| us-west-2      | ami-de08d4be |
-| eu-west-1      | ami-dba2d8a8 |
-| eu-central-1   | ami-81738eee |
-| ap-northeast-1 | ami-4cbb692d |
-| ap-southeast-1 | ami-bb6cc9d8 |
-| ap-southeast-2 | ami-0de1d16e |
+| us-east-1      | ?? |
```
Comment (tnachen) [inline aws/ecs/README.md]: This table is used by the microservice-demo scripts right? Will these "??" be replaced when merged by some script or?

Comment (2opremio) [inline aws/ecs/README.md]: Yes, this PR is WIP :)

File: aws/ecs/cloudformation-larger-app.json (hunk 1 above)
Comment (2opremio) [inline aws/ecs/cloudformation-larger-app.json]: Good catch, thanks.

**Comment by errordeveloper** [comment]
AMI update in CloudFormation templates LGMT.

File: aws/ecs/README.md (hunk 2 above)
Comment (tnachen) [inline aws/ecs/README.md]: Ah sorry didn't see the WIP! :) Anyway I can build AMI and use them myself?

**Comment by 2opremio** [comment]
I generated the new AMIs and tested them (only in us-east-1)

**Pull request merged**